### PR TITLE
fix(network-users): Co-Investigator filter

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: apps/asap-server/src/controllers/teams.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ const priorities: Record<TeamRole, number> = {
   'Project Manager': 3,
   Collaborator: 4,
   'Key Personnel': 5,
+  Advisor: 6,
+  Guest: 7,
+  Staff: 8,
```
Comment (kmaid) [inline apps/asap-server/src/controllers/teams.ts:24]: Took a guess at this

File: packages/model/src/research-output.ts [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import { TeamResponse } from './team';
 import { ListResponse } from './common';
 
-export type ResearchOutputType = 'proposal';
+export type ResearchOutputType =
+  | 'proposal'
```
Comment (kmaid) [inline packages/model/src/research-output.ts:5]: @fampinheiro You said this should match the values shown in squidex which i assume to have proper casing

Comment (fampinheiro) [reply packages/model/src/research-output.ts:5]: On squidex we have proper case :). Can you change it to Proposal?

File: apps/asap-server/src/controllers/teams.ts (hunk 1 above)
Comment (fampinheiro) [reply apps/asap-server/src/controllers/teams.ts:24]: I works, it's not 100% accurate but we can adjust in the future 🙂.

**Review by jeysal — APPROVED** [review]

**Pull request merged**